tests/test_event.py

```diff
@@ -28,6 +57,19 @@ def test_eventview_without_id(self, client):
         response = client.get("/event/view")
         assert response.status_code == status.HTTP_405_METHOD_NOT_ALLOWED
 
+    def test_eventedit_post_correct(self, client, user):
+        response = client.post("/event/edit",
```

Comment:
Can you please change it to url_for?

Reply:
Please try not to mark issues as resolved if you haven't actually solve them.
In other repos it might be considered as a rude behavior.

Reply:
Sorry, I thought I resolved it, my bad.